backend-base/src/main/java/org/eclipse/che/incubator/workspace/telemetry/base/DefaultAnalyticsManager.java

```diff
@@ -42,8 +42,17 @@ public void onActivity() {
   public void onEvent(AnalyticsEvent event, String ownerId, String ip,
       String userAgent, String resolution, Map<String, Object> properties) {
     LOG.info("Event triggered by user {} in {} from ip {} on agent {} :\n{}\nwith resolution: {}\nwith properties:\n{}", getUserId(), ownerId, ip, userAgent, event, resolution, properties);
+    lastEvent = event;
```

Comment:
Shouldn't this be in the `AbstractAnalyticsManager` ? inside `doSendEvent` for example ? It seems the logic that updates the lastXXX properties are really generic.

Reply:
@davidfestal made the requested change